docs/docs/userguide/configuration/CONFIGURATION.md

```diff
@@ -125,14 +126,12 @@ resource_tags = {
 `instance_types` defines the instance type for the EKS cluster node group.
 
 ```terraform
-instance_types = ["m5.2xlarge"]
+instance_types = ["m5.2xlarge"] # (1)
```

Comment:
I think for very important information we should display the warning without need of user click. We may want that users see this warning or any other important messages without hiding them behind a click request.

Reply:
We need to land on a decision on this PR to align ourselves with regards to how to use this docs feature. From my perspective, I think we should hide information that clutters documentation but it can be easily deduced. For example, 

**I would hide under tooltip:**

* Region description - from the name of the variable and the example value, it is quite easy to guess what needs to be there
* Instance type value - I think we can hide the link to the instance types, again, something that is quite easy to deduce

**I wouldn't hide:**

* Warning regarding the static nature of the instance type - if you create a cluster with instance type, you shouldn't know it is a one-way door (at least for now). It shouldn't come as a surprise.
* Values available to the product list, this should be shown at a glance. Users cannot easily deduce it (it is our decision which products we support and forcing them to hover over it, we don't really gain much.

That is my perspective.